versioned/gc/iceberg-actions/pom.xml

```diff
@@ -30,11 +30,20 @@
 
   <name>Nessie - Versioned - GC for Spark - Iceberg - Actions</name>
 
-  <properties>
-    <iceberg.nessie.version>${project.version}</iceberg.nessie.version>
-  </properties>
-
   <dependencies>
+    <dependency>
```

Comment:
why does this have to be added? When it exists in the iceberg runtime?

Reply:
outdated code